tests/plots/test_declarative.py

```diff
@@ -1184,3 +1184,22 @@ def test_panel():
 
     pc.panel = panel
     assert pc.panel is panel
+
+
+def test_copy():
+    """Test that the copy method works for all classes in `declarative.py`."""
+    # Copies of objects should not point to the same location in memory
+    objects = [ImagePlot(), ContourPlot(), FilledContourPlot(), BarbPlot(),
+               MapPanel(), PanelContainer(), PlotObs()]
+
+    for obj in objects:
+        copied_obj = obj.copy()
+        assert obj != copied_obj
```

Comment:
I'm kinda surprised this actually fails. If you're trying to check that they're not the same object in memory, us `is not`:
```suggestion
        assert obj is not copied_obj
```